### PR TITLE
[GTM] Resource prices swagger preview version

### PR DESCRIPTION
**Pull request by kjeur**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.

### CODE REVIEW

**Comment by kjeur** [comment]
@anuchandy who can help with ARM review?

**Comment by kjeur** [comment]
@ravbhatnagar Can you please help me here?

**Comment by anuchandy** [comment]
@kjeur we've requested review from ARM team.

**Comment by kjeur** [comment]
@ravbhatnagar any update here?

**Comment by ravbhatnagar** [comment]
@vladimirjoanovic for ARM review

File: specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {  
+    "/providers/Microsoft.Billing/subscriptions/{subscriptionId}/pricesheet/resourceprices": {
+      "post": {
```
[the remark this answers is not in the document]
Comment (kjeur) [reply specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json]: Resourceprice is extension of pricesheet. Will introduce pricesheet in next week.

File: specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json [hunk 2]
```diff
@@ -0,0 +1,255 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-11-01-preview",
+    "title": "BillingManagementClient",
```
Comment (vladimirjoanovic) [inline specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json]: normally when a new version of an API is created the existing resources/operations are exposed from the most recent preview are also included and not only the new stuff.  Is it intended to not include content from https://github.com/Azure/azure-rest-api-specs/blob/master/specification/billing/resource-manager/Microsoft.Billing/preview/2018-08-01-preview/billing.json

File: specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource name."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource type."
```
Comment (vladimirjoanovic) [inline specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json]: is there a standard list of resource types that will be exposed? ie. does it include RP name and resource type name or is still a billing type id?

Comment (kjeur) [reply specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json]: This include RP/pricesheet/resourceprices

File: specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json (hunk 2 above)
Comment (kjeur) [reply specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json]: Agreed.. Currently with this new version we have spec ready for just resourceprices. Rest all stuff we are still working on finalizing contract.

**Comment by kjeur** [comment]
@ravbhatnagar any help here?

**Comment by ravbhatnagar** [comment]
@kjeur - I think it makes more sense to review both pricesheets and resourcePrices together. Currently, without specific details on how pricesheets is modeled, it wont be possible to do a good review on resourcePrices.

**Comment by ravbhatnagar** [comment]
Since these are new types being added, please feel free to set up a meeting as noted in the process here -
https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html

**Comment by ravbhatnagar** [comment]
Does this still need ARM review? @anuchandy - This PR has been active for a long time. @kjeur - please close this if its not needed.

**Comment by salameer** [comment]
@kjeur and @anuchandy 

I'll be closing this by tomorrow if there's no action on this :) its been open for a while now :)

**Comment by anuchandy** [comment]
@kjeur closing this due to no action.

**Pull request closed without merging**